requirements.txt

```diff
@@ -1,2 +1,3 @@
 git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/XBlock.git@cdyer/scorable-mixin
```

Comment:
PR to upgrade xblock-utils: https://github.com/edx/xblock-utils/pull/42

Not sure why I just got all those geckodriver-related failures, though...